ionic_user_interface/src/providers.ts

```diff
@@ -1,42 +1,96 @@
 import { ref, Ref } from 'vue';
 import axios from 'axios';
 import router from '@/router';
+import jwt_decode from 'jwt-decode';
+import { toastController } from '@ionic/vue';
 
 const isLoggedIn = ref(false);
 const userEmail = ref('');
 const accessToken = ref('');
 const idToken = ref('');
 const isConfirmationNeeded = ref(false);
+const prefersDarkMode = ref(false);
+
+const forceLogout = async (): Promise<void> => {
+  const config = {
+    headers: {
+      Authorization: `Bearer ${accessToken.value}`,
+    },
+  };
+  return axios
+    .post(process.env.VUE_APP_USER_ENDPOINT_URL + '/user/logout', {}, config)
+    .then((response) => {
+      console.log(response.data);
+    })
+    .catch((error) => {
+      if (error.response) {
+        // The request was made and the server responded with a status code
+        // that falls out of the range of 2xx
+        console.log(error.response.data);
```

Comment:
You can leave the comments if you want but maybe reduce the num of log lines and change them to console.error just to be more explicit